bootloader/bootloader.go

```diff
@@ -65,7 +65,7 @@ type installableBootloader interface {
 // InstallBootConfig installs the bootloader config from the gadget
 // snap dir into the right place.
 func InstallBootConfig(gadgetDir, rootDir string) error {
-	for _, bl := range []installableBootloader{&grub{}, &uboot{}, &androidboot{}, &lk{}} {
+	for _, bl := range []installableBootloader{&grub{}, &recoveryGrub{}, &uboot{}, &androidboot{}, &lk{}} {
```

Comment:
this change looks strange, we want to install only one of grub or recoveryGrub per a given rootdir, the gadget will define both or just the run one (a bit to discuss how we want to do that) ?